lib/Payment.ts

```diff
@@ -8,6 +8,7 @@ import * as types from "./types";
  */
 export class Payment {
   id: string = "";
+  externalId: string = "";
```

Comment:
I added this so that I can test upon it; lmk if I should revert and test something else

Reply:
Yeah lets remove it once we are done testing, so that this class keeps mimicking the publicly available objects' lists